### PR TITLE
Ad drop down

### PR DESCRIPTION
**Pull request by alice-dempsey**
I did the thing!

### CODE REVIEW

**Comment by wolfep15** [comment]
Awesome Alice! 

Something I've run into is it only seems to suggest names that are in the displayed list, and not the entire array. 

For example, if I've typed "A", and the autocomplete suggestion thing is displaying 6 names such as:
Aaron
Abel
Adam
Adrian
Alan
Arthur

If I type "Az", it won't display a name like "Azarel", even if "Azarel" is in the array that it originally pulled from.
 
It's totally possible that I'm just testing this in an incorrect way, if you could test it on your set up and let me know if it works for you (or if this doesn't make sense).

**Comment by alice-dempsey** [comment]
Ok! I'll work on fixing that this weekend.

File: src/style/style.css [hunk 1]
```diff
@@ -45,7 +45,11 @@ body {
 }
 
 .one_btn{
-    margin-top: 15%;
+/*PROBLEM: THIS POSITION ISNT WORKING, NEED TO HAVE IT STAY IN ONE PLACE*/
```
Comment (wolfep15) [inline src/style/style.css]: Is this still a problem?

Comment (wolfep15) [reply src/style/style.css]: Going over this again, realized we talked about this on Sunday and decided it's fine. Merging now, great work!!!!

Comment (wolfep15) [reply src/style/style.css]: Going over this again, realized we talked about this on Sunday and decided it's fine. Merging now, great work!!!!

**Pull request merged**